versioning-and-branching.md

```diff
@@ -60,6 +60,8 @@ git remote add upstream git@github.com:usnistgov/OSCAL.git
 
 ## Personal Working Branches
 
+Staff on the NIST OSCAL Team can do work on branches in this repository or choose to do work as an individual outside contributor in a fork as described below.
+
 All individual work will be done in branches in a personal fork of this repository.
```

Comment:
Suggestion: "All individual work must be done in branches named per guidance below, and in a personal fork of this repository."

Reply:
Ditto my same concern over your feedback: it means I do have to make a changes to not imply directly or forking implies whether it is as employee or outside work. The feedback definitely makes a good point, I will adjust here too.

Reply:
Oh interesting, this seems to also be a dupe of the one above, not separate, because the additional space changes the content of the diff maybe so it is repeated and not considered identical. In any event, changes forthcoming.